actionview/test/template/log_subscriber_test.rb

```diff
@@ -63,23 +70,62 @@ def test_render_inline_template
     end
   end
 
-  def test_render_partial_template
+  def test_render_partial_with_implicit_path
     Rails.stub(:root, File.expand_path(FIXTURE_LOAD_PATH)) do
-      @view.render(:partial => "test/customer")
+      @view.render(Customer.new("david"), :greeting => "hi")
       wait
 
       assert_equal 1, @logger.logged(:info).size
-      assert_match(/Rendered test\/_customer.erb/, @logger.logged(:info).last)
+      assert_match(/Rendered customers\/_customer\.html\.erb/, @logger.logged(:info).last)
```

Comment:
Because this is testing if we pass a customer object and the renderer can still find the right template. I think we don't cover this kind of situation in our new tests.